client/sprotty-ecore/css/diagram.css

```diff
@@ -1,10 +1,21 @@
+:root {
+    --sprotty-background: var(--theia-layout-color3);
+    --sprotty-edge: var(--theia-ui-font-color1);
+    --sprotty-border: var(--theia-border-color2);
+}
+
 svg {
     margin-top: 15px;
     width: 100%;
     height: 500px;
     border-style: solid;
     border-width: 1px;
-    border-color: #bbb;
+    border-color: var(--sprotty-border);
+}
+
+.label-edit input {
+    background: rgba(255, 255, 255, 0.5);
```

Comment:
doesn't this look weird in dark theme? I didn't test it, but color wise this looks weird.

Reply:
I attached a screenshot in the description. I think it looks fine.